cl/settings/third_party/celery.py

```diff
@@ -6,6 +6,21 @@
 DEVELOPMENT = env.bool("DEVELOPMENT", default=True)
 CELERY_ETL_TASK_QUEUE = env("CELERY_ETL_TASK_QUEUE", default="celery")
 CELERY_IQUERY_QUEUE = env("CELERY_IQUERY_QUEUE", default="celery")
+CELERY_QUEUES = (
+    "batch0",
```

Comment:
It seems that we're missing the `celery` queue?

```suggestion
    "celery",
    "batch0",
```